### PR TITLE
Add "Do you have a history of addiction?" question

### PR DESCRIPTION
**Pull request by brucebolt**
Adds the "Do you have a history of addiction?" question, with placeholder next and previous locations.

Screenshot 2020-03-21 at 15 10 37

Trello card: https://trello.com/c/gxj9zkcc

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -37,3 +37,10 @@ en:
           label: Yes, I've been tested and I do not have coronavirus
         option_no:
           label: No, I have not been tested
+    addiction:
+      title: Do you have a history of addiction?
+      options:
+        option_yes:
+          label: Yes, I’ve had problems with drug addiction or alcohol addiction (alcoholism)
+        option_no:
+          label: "No"
```
Comment (issyl0) [inline config/locales/en.yml:67]: Can we be consistent about the use of quotes or not in these options? Some have them, some don't, this one is inconsistent in itself...

**Review by issyl0 — COMMENTED** [review]
LGTM. One YAML style nitpick but it shouldn't block shipping this, it's just potentially confusing to devs.

**Review by issyl0 — APPROVED** [review]
And I meant the above to be an approval...

File: config/locales/en.yml (hunk 1 above)
Comment (brucebolt) [reply config/locales/en.yml:67]: My preference is to not have them, but "No" is a reserved word, so needs the quotes.  Maybe we should just quote everything?

**Pull request merged**